src/components/FeatureViews/Accounts/ComponentViewer.js

```diff
@@ -13,7 +13,14 @@ import StudentCourseViewer from "./TabComponents/StudentCourseViewer";
 import StudentInfo from "./TabComponents/StudentInfo";
 
 const ComponentViewer = (props) => {
+    const [onTransition, setOnTransition] = useState(false);
+
+    useEffect(()=>{
+        setOnTransition(true);
+
+    },[props]);
```

Comment:
What is this for?

Reply:
lol was trying to get page transitions to work, but something weird with the loading